core/examples/luigi-sample-angular/src/assets/sampleconfig.js

```diff
@@ -1,5 +1,21 @@
-var getAllProjects = function() {
-  return new Promise(function(resolve) {
+var navigationPermissionChecker = function (nodeToCheckPermissionFor, parentNode, currentContext) {
```

Comment:
In navigation.js you use this function with variables _currentPath, currentNavigationStructure, node_ - could you use similar naming in both places? Because at the beginning I wasn't sure what this function uses 🙂

Reply:
done

Reply:
Great :) But I saw you changed arrow functions to use ES5 - could you also remove this one 'const'  used here since it's not ES5 compatible as well?